files/etc/ublue-update.d/system/00-system-update.sh

```diff
@@ -3,15 +3,27 @@
 check_for_rebase() {
     IMAGE_REF_FILE="$1"
     if [ -f "$IMAGE_REF_FILE" ]; then
-        LOCAL_IMAGE_REF=$(rpm-ostree status --pending-exit-77 -b --json | jq -r '.deployments[0]["container-image-reference"]')
-        if [[ "$LOCAL_IMAGE_REF" == "null" ]]; then
+
+        STATUS_COMMAND="rpm-ostree status --pending-exit-77 -b --json"
+        LOCAL_IMAGE_REF=$($STATUS_COMMAND | jq -r '.deployments[0]["container-image-reference"]')
+
+        if ! $STATUS_COMMAND >/dev/null || [[ "$LOCAL_IMAGE_REF" == "null" ]]; then
+            # if jq failed to get the variable, or rpm-ostree had a nonzero exit code, return
             return
         fi
+
+        LOCAL_IMAGE_PREFIX=$(echo "$LOCAL_IMAGE_REF" | awk -F ":" '{print $1}')
         LOCAL_IMAGE_REF_UNTAGGED=$(echo "$LOCAL_IMAGE_REF" | awk -F ":" '{print $1":"$2":"$3}')
```

Comment:
```suggestion
        LOCAL_IMAGE_REF_UNTAGGED=$(echo "$LOCAL_IMAGE_REF" | awk -F ":" '{print $1":"$2}')
```

Reply:
Old code will get you 
```
++ echo ostree-unverified-registry:ghcr.io/ublue-os/bluefin-dx:38
++ awk -F : '{print $1":"$2":"$3}'
+ LOCAL_IMAGE_REF_UNTAGGED=ostree-unverified-registry:ghcr.io/ublue-os/bluefin-dx:38
```

New code will get you 
```
++ echo ostree-unverified-registry:ghcr.io/ublue-os/bluefin-dx:38
++ awk -F : '{print $1":"$2}'
+ LOCAL_IMAGE_REF_UNTAGGED=ostree-unverified-registry:ghcr.io/ublue-os/bluefin-dx
```

Reply:
should account for the shorthand now